ProgrammingInC.md

```diff
@@ -0,0 +1,43 @@
+**[Back](/README.md/)**
+
+# Programming in C
+
+## Videos
+
+- [Basic C Programming](https://www.youtube.com/watch?v=rLf3jnHxSmU&list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
+
+- [C Programming & Data Structures](https://www.youtube.com/watch?v=4OGMB4Fhh50&list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM)
+
+- [C Programming Edureka](https://www.youtube.com/watch?v=8PopR3x-VMY)
+
+## Git Books
+
+- [Data Structures in C](https://nitinranganath.gitbook.io/data-structures/)
+
+## Books
+
+- [Introduction to Algorithms](https://edutechlearners.com/download/Introduction_to_algorithms-3rd%20Edition.pdf) by Thomas H. Cormen, Charles E. Leiserson, Ronald L. Rivest, and Clifford Stein
+
+- [Competitive Programming 3](http://www.sso.sy/sites/default/files/competitive%20programming%203_1.pdf) by Steven Halim and Felix Halim
+
+- [Competitive Programmers Hand Book](https://cses.fi/book/book.pdf) Beginner friendly hand book for competitive programmers.
+
+- [Data Structures and Algorithms Made Easy](https://github.com/Amchuz/My-Data-Structures-and-Algorithms-Resources/raw/master/Books/Data%20Structures%20and%20Algorithms%20-%20Narasimha%20Karumanchi.pdf) by Narasimha Karumanchi
+
+- [Learning Algorithms Through Programming and Puzzle Solving](https://github.com/Amchuz/My-Data-Structures-and-Algorithms-Resources/raw/master/Books/Learning%20Algorithms%20Through%20Programming%20and%20Puzzle%20Solving.pdf) by Alexander Kulikov and Pavel Pevzner
+
+## Courses
+
+- [C Programming for Biginners](https://www.udemy.com/course/c-programming-for-beginners-/)
+
+- [Microcontroller Embedded C Programming: absolute beginners](https://www.udemy.com/course/microcontroller-embedded-c-programming/)
+
+- [Fork CPP](https://www.geeksforgeeks.org/fork-cpp-course-structure) <h1>A good course for beginners</h1>
+
+- [EDU](https://codeforces.com/edu/course/2) Advanced course
+
+- [C++ For Programmers](https://www.udacity.com/course/c-for-programmers--ud210) Learn features and constructs for C++
+
+### Free Mentor
```

Comment:
Remove this  no-self promotion is allowed

Reply:
I did it @arjunkharbanda